tools/sof-kernel-log-check.sh

```diff
@@ -276,6 +276,9 @@ case "$platform" in
         # https://unix.stackexchange.com/questions/109294/mei-00000016-0-init-hw-failure
         ignore_str="$ignore_str"'|mei_me 0000:00:16\..: wait hw ready failed'
         ;;
+    ehl)
+        # https://github.com/thesofproject/sof-test/issues/695
+        ignore_str="$ignore_str"'|device-mapper: table: 253:0: thin-pool: unknown target type'
```

Comment:
Not sure if "253:0" will be changed in future, better use RegExp.

Reply:
Updated